test/OrganisationRegistry.SqlServer.IntegrationTests/OnProjections/Body/WhenScheduledCommandsAreRun/Given_some_activeBodyOrganisation.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace OrganisationRegistry.SqlServer.IntegrationTests.OnProjections.Body.WhenScheduledCommandsAreRun
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AutoFixture;
+    using FluentAssertions;
+    using OrganisationRegistry.Body;
+    using OrganisationRegistry.Body.Commands;
+    using OrganisationRegistry.Infrastructure.Commands;
+    using SqlServer.Body.ScheduledActions.Organisation;
+    using Xunit;
+
+    [Collection(SqlServerTestsCollection.Name)]
+    public class Given_some_activeBodyOrganisation
```

Comment:
Weird capitalization?

Reply:
Given_some_activeBodyOrganisationS

Reply:
liever caps na een underscore, of geen caps ?

Reply:
ben gegaan voor caps na underscore